vignettes/covidHubUtils-overview.Rmd

```diff
@@ -118,6 +175,41 @@ p + scale_x_date(name=NULL, date_breaks = "1 month", date_labels = "%b") +
 Additionally, `plot_forecasts()` can handle plotting multiple models or locations or forecast dates at the same time as the following examples show.
 
 
+## as_of versioning
+
+```{r message=FALSE}
+# Load forecasts that were submitted in a time window from zoltar
+```
+
+## Hub Location
+
```

Comment:
Maybe add a paragraph saying we support query for "European COVID-19 Forecast Hub"?

Reply:
I have included this